src/www/index.html

```diff
@@ -101,7 +107,7 @@ <h2 class="text-sm text-gray-400 mb-10"></h2>
                 <div class="flex-grow">
 
                   <!-- Name -->
-                  <div class="text-gray-700 group" :title="'Created at ' + dateTime(new Date(client.createdAt))">
+                  <div class="text-gray-700 group" :title="'Created on ' + dateTime(new Date(client.createdAt))">
```

Comment:
Why has this changed? `at` is used for exact times, `on` for days/months/years.

Reply:
That's why it's changed:

![image](https://user-images.githubusercontent.com/24798198/169423981-7c5a78f3-f37a-43df-af4b-8dccaf8065e1.png)

I'm not sure if it is displayed in a different format for you, in which case you would want to handle the format order.